.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+TODO:
+
+- [ ] ‹fill in›
+
+<!-- notes for reviewers -->
+
+<!-- Links to other issues or pull requests,
+     for cross-repository links use: ‹namespace›/‹repository›#‹ID of issue›
+       (‹namespace›/‹repository›!‹ID of PR› respectively)
+-->
+
+Fixes
+
+Related to
+
+Merge before/after
+
+---
+
+<!-- release notes for blog follow -->
```

Comment:
Ad-hoc contributors might be confused about what/where/how the blog is, while changelog is pretty straightforward.
And we'll reuse it in both.

```suggestion
<!-- release notes for changelog/blog follow -->
```

Reply:
thanks, will update in all :)